cmd/globals.go

```diff
@@ -464,6 +464,9 @@ var (
 	globalDynamicAPIPort bool
 
 	// Add new variable global values here.
+
+	// We expect initially that all would go through
+	globalIAMFullyInitialized = true
```

Comment:
You sure you don't want an `atomic.Bool`?

Reply:
hmm, let me try that